test/functional/test-integration.js

```diff
@@ -34,43 +34,33 @@ describe('3p integration.js', () => {
   });
 
   it('should register integrations', () => {
-    expect(registrations).to.include.key('a9');
-    expect(registrations).to.include.key('adblade');
-    expect(registrations).to.include.key('adition');
-    expect(registrations).to.include.key('adform');
+    // Get all ad extensions from test env.
+    const extensions = window.ampTestRuntimeConfig.adExtensions;
```

Comment:
This test case is to catch any unregistered ad networks. However, if people forgot register the ad network at the first place, he/she will not update this test case as well. So we end up not catching any violation.